Wiki/NewToFvwm/WhyFvwm/index.md

```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
 popular window manager, and is now a tiny project still used
 by lots of core users. What is it about fvwm that users like?
 
-+ Fvwm is light weight and fast. Fvwm is built on top of X11,
+{% capture fvwmtxt %}
+<p><strong>Fvwm is light weight and fast.</strong> - Fvwm is built on top of X11,
```

Comment:
Remind me again why we need this HTML markup here?  Can't we do this just with Markdown?

```
**Fvwm is lightweight and fast**
```

Also, I would correct "light weight" to "lightweight".